src/Classes/Utils/Utils.ts

```diff
@@ -84,45 +135,37 @@ export class Utils {
         return velocityBase;
     }
 
-    public static getParticleColor(options: IOptions, color: { value: string[] | IColor | string }): IColor {
-        const res: IColor = {};
+    public static getParticleColor(options: IOptions, color: { value: string[] | IColor | string }): IRgb | null {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public static method not allowed after declaration of private static method. Instead, this should come at the beginning of the class/interface.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)